pkg/config/decode.go

```diff
@@ -48,10 +48,11 @@ type FromStringer interface {
 //
 // Most of the heavy lifting is handled by the mapstructure library. A custom decoder is used to handle
 // decoding string values to the supported primitives.
-func Decode(input interface{}, output interface{}) error {
+func Decode(input interface{}, output interface{}, defaults ...interface{}) error {
```

Comment:
`defaults ...interface{}` can be removed now, correct?

Reply:
Yep, it's gone now.